src/aws-resource-mappings.ts

```diff
@@ -62,6 +63,14 @@ export function mapToAwsResource(
                 },
                 options,
             );
+            return {
+                resource,
+                attributes: {
+                    ...getAttributesFromResource(resource),
+                    id: resource.id,
```

Comment:
It looks like ApiGatewayV2::Integration shows up in the metadata.json now so we need to override which `id` to use.

I'll create an issue to track this. If a user maps something to an aws resource and then later it becomes available in CCAPI we don't want it to cause a failure like this. We need to somehow exclude ones that have been tracked.

Reply:
Hmm what was the failure? When I was auditing it, the metadata selection matched what was in the code; moreover `id` should be respected here ahead of what's written in metadata.

Reply:
Yes with this update (adding `id` to `attributes`) we are able to override the behavior. The issue here is that this particular resource was added to CCAPI so it now shows up with a `Ref` mapping of `integrationId` which is not an attribute available on the AWS provider resource.